onnxmltools/convert/common/_apply_operation.py

```diff
@@ -55,15 +53,24 @@ def _apply_basic_numerical_operation(scope, op_type, input_names, output_name, c
     container.add_node(op_type, input_names, output_name, op_version=op_version, name=name, **attrs)
 
 
-def _apply_pointwise_operation(scope, op_type, input_names, output_name, container, operator_name):
+def _apply_pointwise_operation(scope, op_type, input_names, output_name, container, operator_name, broadcast):
     name = _create_name_or_use_existing_one(scope, op_type, operator_name)
 
     attrs = {}
-    if container.target_opset < 6:
-        attrs['consumed_inputs'] = [0] * len(input_names)
-        op_version = 1
+
+    # Before ONNX-1.3 (opset 8), broadcasting behavior is Caffe2-like.
+    if container.target_opset < 8:
+        if container.target_opset < 6:
+            attrs['consumed_inputs'] = [0] * len(input_names)
+            op_version = 1
+        else:
+            op_version = 6
+
+        if broadcast is not None:
+            attrs['broadcast'] = broadcast
```

Comment:
Hi @wenbingl! I think opset 7 broadcasting support is for the basic_numerical_operations (i.e. Add, Div, Mul, Pow, Sub)... I think the pointwise operations were only updated in opset 8 (i.e. Mean, Max, Min, Sum). Can you confirm?

See line 32.

` def _apply_basic_numerical_operation(scope, op_type, input_names, output_name, container, operator_name,
                                      axis, broadcast):
     name = _create_name_or_use_existing_one(scope, op_type, operator_name)
 
     attrs = {}
     if container.target_opset < 7:
         # Before ONNX-1.2 (opset 7), broadcasting behavior is Caffe2-like.
         if axis is not None:
             attrs['axis'] = axis
         if broadcast is not None:
             attrs['broadcast'] = broadcast
 
         if container.target_opset < 6:
             attrs['consumed_inputs'] = [0, 0]
             op_version = 1
         else:
             op_version = 6
     else:
         # Since ONNX-1.2 (opset 7), broadcasting behavior is Numpy-like, so we don't need to specify any attributes
         op_version = 7
 
    container.add_node(op_type, input_names, output_name, op_version=op_version, name=name, **attrs)`

Reply:
specifically, I didn't find any attribute in Max/Mean/Min named 'broadcast',  Did you find anyone? https://github.com/onnx/onnx/blob/master/docs/Operators.md#mean

Reply:
I see! I misunderstood the API! Thanks for the catch. See [most recent update](https://github.com/onnx/onnxmltools/pull/171/commits/6d8d31ab1f296f5409c98774ea2b75034875922d).